spec/factories/birth_record.rb

```diff
@@ -2,7 +2,7 @@
 
 FactoryBot.define do
   factory :birth_record do
-    sequence(:first_and_middle_names) { |n| "#{Faker::Name.first_name} #{n}" }
+    first_and_middle_names { Faker::Name.name_with_middle }
```

Comment:
this makes a 3 name long name. How about using Name.middle instead?

Reply:
Done